src/utils.py

```diff
@@ -43,23 +43,46 @@ def get_events():
     return events
 
 
-def sync_organization():
-    pass
+# TODO
+# sync_members()
+
+
+def sync_customer(project_id):
+    project_data = mp.get_project(project_id=project_id)
+    for customer in wc.list_customers():
```

Comment:
Why not simply use filtering `wc.list_customers({'name': project_data.attributes.organization})` instead of cycle?